tendrl/ceph_integration/manager/crud.py

```diff
@@ -32,13 +31,31 @@ def __getattribute__(self, item):
             attr = object.__getattribute__(self, item)
             if callable(attr):
                 def wrap(*args, **kwargs):
-                    LOG.debug("Crud >> %s(%s, %s)" %
-                              (item, args, kwargs))
+                    try:
+                        event.Event(Message(
+                            Message.priorities.DEBUG,
+                            Message.publishers.CEPH_INTEGRATION,
+                            {"message": "Crud >> %s(%s, %s)" %
+                                (item, args, kwargs)}))
+                    except event.EventFailed:
+                        print(traceback.format_exc())
```

Comment:
@r0h4n This will give error when socket failed, if socket failed we cant log anything, so we cant log it